hack/update-staging-godeps-dockerized.sh

```diff
@@ -51,6 +51,7 @@ while getopts ":df" opt; do
 done
 
 kube::util::ensure_dockerized
+KUBE_GOPATH="/godeps"
```

Comment:
This is used here: https://github.com/kubernetes/kubernetes/pull/50717/files#diff-dac3f0d616ae45f98d45d6420ed1be65R19 so that `kube::golang::setup_env` sets up the deps in that path